pkg/client/actionclient_test.go

```diff
@@ -51,6 +51,7 @@ var _ = Describe("ActionClient", func() {
 	var (
 		rm meta.RESTMapper
 	)
+	const mockTestDesc = "Test Description"
```

Comment:
I see two more usages of "Test Description" in this file (in `verifyRelease()` and `verifyNoRelease()`). I think we can make this a global constant and use this in those functions as well.